src/nextjournal/clerk/sci_viewer.cljs

```diff
@@ -1165,7 +715,3 @@ black")}]))}
   (sci/eval-form @!sci-ctx f))
 
 (set! *eval* eval-form)
-
-(swap! viewer/!viewers (fn [viewers]
-                         (-> (into {} (map (juxt key (comp #(into [] (map viewer/process-render-fn) %)  val))) viewers)
-                             (update :root concat js-viewers))))
```

Comment:
`js-viewers` were placed at key `:root` and hence unused, and I'm not sure we need to process render fns as before at all.

Reply:
Render fns are "processed" as a result of the presentation of a value.